safe/impact_function/impact_function.py

```diff
@@ -2695,6 +2717,8 @@ def load_from_output_metadata(output_metadata):
         else:
             impact_function._crs = None
 
+        impact_function._provenance_ready = True
```

Comment:
Where was this statement before?

Reply:
it wasn't anywhere there before. previously  this method doesn't set the provenance for the IF, but I need the IF to have provenances to make t works with the reporting engine, so I added this line as well as the provenance